assets/dev/js/editor/views/preview.js

```diff
@@ -71,16 +71,12 @@ const Preview = BaseSectionsContainerView.extend( {
 	},
 
 	createElementFromModel: function( model, options = {} ) {
-		const shouldWrap = ! [ 'container', 'section' ].includes( model.elType || model?.get( 'elType' ) );
-
-		if ( ! shouldWrap ) {
-			delete options.at;
-		}
+		delete options.at;
```

Comment:
Why?

Reply:
If a json template is dragged (imported) into the document, and it contains sections with their `at` index, we want to ignore it and instead put the template in the position in which it was dragged into.

Reply:
In which case the `at` is saved to the JSON?

Reply:
@matipojo Sorry, I was wrong, it's not saved to JSON.
I removed this line.